src/Microsoft.Health.Dicom.Core/Features/Store/FailureReasonCodes.cs

```diff
@@ -34,5 +34,13 @@ internal static class FailureReasonCodes
         /// The DICOM instance is being created.
         /// </summary>
         public const ushort PendingSopInstance = 45071;
+
+        /// <summary>
+        /// Data Set does not match SOP Class
+        /// </summary>
+        /// <remarks>
+        /// The Studies Store Transaction (Section 10.5) observed that the Data Set did not match the constraints of the SOP Class during storage of the instance.
+        /// </remarks>
+        public const ushort DataSetDoesNotMatchSOPClass = 45063;
```

Comment:
Could you update the Conformance Statement

Reply:
Done.